python/batch-predictor/main.py

```diff
@@ -122,7 +122,7 @@ def main(spec_path, spark):
             .builder \
             .appName(args.job_name) \
             .getOrCreate()
-        spark.conf.set("spark.sql.execution.arrow.enabled", "true")
+        spark.conf.set("spark.sql.execution.arrow.pyspark.enabled", "true")
```

Comment:
This is a small fix since `spark.sql.execution.arrow.enabled` option is now deprecated.